File: test/cs/case20_fstring.cs
// このファイルは `test/cs/case22_fstring.cs` の変換結果（C#サンプル）です。
// Python入力との対応関係を追いやすくするため、日本語コメントを付与しています。
// 仕様変更時は、対応する Python 側ケースと合わせて更新してください。

using System;

public static class Program
{
    public static string make_msg_22(string name, int count)
    {
        return $"{name}:22:{count}";
    }

    public static void Main(string[] args)
    {
        Console.WriteLine(make_msg_22("user", 7));
    }
}

File: test/cpp/case20_fstring.cpp
#include "cpp_module/py_runtime.h"

// このファイルは `test/py/case20_fstring.py` のテスト/実装コードです。
// 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
// 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

str make_msg_22(str name, int64 count) {
    return (py_to_string(name) + ":22:" + py_to_string(count));
}

int main() {
    py_print(make_msg_22("user", 7));
    return 0;
}
